benchmarks/dynamo/common.py

```diff
@@ -432,6 +446,12 @@ def maybe_mark_profile(*args, **kwargs):
         else:
             yield
 
+    times = args.iterations_per_run
+
+    # Use higher tolerance for XLA since XLA cause numerical unstability when
+    # graph size changes.
+    tolerance = 1e-2 if args.trace_on_xla else 1e-4
```

Comment:
Let's make this command line flag as well, if we are doing comparative benchmarking we want to make sure both sides of the experiment are held to the same standard.

Reply:
yep, all our experiments are using the same tolerance since they are all 'trace_on_xla'.